<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>继承QThread线程</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnthreadRun">
           <property name="text">
            <string>QThread Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnthreadLocalRun">
           <property name="text">
            <string>QThread Run Local</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btngetSomthing">
           <property name="text">
            <string>getSomthing</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnthreadQuit">
           <property name="text">
            <string>QThread Quit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnsetSomething">
           <property name="text">
            <string>setSomething</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnthreadTerminate">
           <property name="text">
            <string>QThread Terminate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btndoSomthing">
           <property name="text">
            <string>doSomthing</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>继承QOjbect线程</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btnObjectMove2Thread">
           <property name="text">
            <string>ObjectMove2Thread</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnThreadStop">
           <property name="text">
            <string>ThreadStop</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btnObjectMove2Thread2">
           <property name="text">
            <string>ObjectMove2Thread2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnQuit">
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>HeartBeat</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBarheart">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
